### DISCUSSION TITLE
Shell Operator YouTube Video

### DISCUSSION THREAD

**Discussion by Piotr1215** [flant/shell-operator#626] [Show and tell]
[Shell operator](https://flant.github.io/shell-operator/) is a unique project. It's more of a building block or a tool for platform builders.

I've put together a video showing how to use shell operator and some other projects to automate pod lifecycle management with Slack notifications:

https://www.youtube.com/watch?v=murezpqn5yI

**Comment by nabokihms** [comment]
This is amazing, @Piotr1215. Thank you for sharing 🙌
